optuna/storages/_grpc/client.py

```diff
@@ -67,28 +73,35 @@ class GrpcStorageProxy(BaseStorage):
     """
 
     def __init__(self, *, host: str = "localhost", port: int = 13000) -> None:
-        self._stub = api_pb2_grpc.StorageServiceStub(
-            grpc.insecure_channel(
-                f"{host}:{port}",
-                options=[("grpc.max_receive_message_length", -1)],
-            )
-        )  # type: ignore
-        self._cache = GrpcClientCache(self._stub)
         self._host = host
         self._port = port
+        self.setup()
+
+    def setup(self) -> None:
+        self._channel = create_insecure_channel(self._host, self._port)
+        self._stub = api_pb2_grpc.StorageServiceStub(self._channel)
+        self._cache = GrpcClientCache(self._stub)
+
+    def wait_server_ready(self, timeout: float) -> None:
+        try:
+            with create_insecure_channel(self._host, self._port) as channel:
+                grpc.channel_ready_future(channel).result(timeout=timeout)
+        except grpc.FutureTimeoutError as e:
+            raise ConnectionError("GRPC connection timeout") from e
+
+    def close(self) -> None:
```

Comment:
Currently, the docstrings for the `close`, `setup`, `wait_server_ready` methods are empty as shown in the link below. Could you add docstrings?

https://optuna--6010.org.readthedocs.build/en/6010/reference/generated/optuna.storages.GrpcStorageProxy.html#optuna.storages.GrpcStorageProxy